web/src/__tests__/footer.test.tsx

```diff
@@ -0,0 +1,23 @@
+import { render, screen } from '@testing-library/react'
+
+import { Footer } from '../components/Footer'
+
+describe('Vercel Logo and Link', () => {
+  it('renders the corrcet logo', () => {
```

Comment:
Typo!

Reply:
Thanks — in the future, with minor typos like these, please just commit the suggestion directly straight from the GitHub UI

Example:
```suggestion
  it('renders the correct logo', () => {
```

Reply:
<img width="750" alt="Screen Shot 2021-04-05 at 9 23 19 PM" src="https://user-images.githubusercontent.com/29822597/113658612-3164ac80-9655-11eb-95cb-d0bbd6f4ad61.png">